.github/workflows/pr.yml

```diff
@@ -46,12 +46,17 @@ jobs:
     strategy:
       matrix:
         k8s: [v1.17.11, v1.18.8, v1.19.4, v1.20.0]
+        rabbitmq-image:
+        - rabbitmq:3.8.8-management
+        - rabbitmq:3.8.9-management
+        - rabbitmq:3.8.10-rc.5-management
```

Comment:
`3.8-rc-management` could be a better tag because there might be `rc.6` in the future.